hack/boilerplate/boilerplate.py

```diff
@@ -36,7 +36,7 @@
 parser.add_argument(
     "--rootdir", default=rootdir, help="root directory to examine")
 
-default_boilerplate_dir = os.path.join(rootdir, "hack/boilerplate")
+default_boilerplate_dir = os.path.join(rootdir, "vendor/k8s.io/code-generator/hack")
```

Comment:
Is k8s.io/code-generator kubernetes specific?

I would have expected each repo to have its own boilerplate, so that the code generator can be used in any context. Changing the main repo to use boilerplate from the code-generator repo doesn't seem right.

Reply:
👍